static-src/scripts/maps/state/constants.js

```diff
@@ -19,4 +19,5 @@ export const TILE_LAYER_TYPE = 'tiles'
 *   Boundaries of San Francisco for map defaults.
 */
 
-export const SAN_FRANCISCO_BOUNDS = [ [ 37.813996, -122.529439 ], [ 37.702302, -122.348852 ] ]
+export const SAN_FRANCISCO_BOUNDS = [ [ 37.833996, -122.539439 ], [ 37.682302, -122.278852 ] ]
```

Comment:
This does help with the problem a little: it looks like before, the map wasn't initially centered on San Francisco:
<img width="1383" alt="Screen Shot 2019-07-22 at 1 43 45 PM" src="https://user-images.githubusercontent.com/13007381/61664053-d58ea000-ac86-11e9-9a5f-785c8cc6fd8b.png">
and now it's moved over so SF is centered:
<img width="1375" alt="Screen Shot 2019-07-22 at 1 44 02 PM" src="https://user-images.githubusercontent.com/13007381/61664072-dfb09e80-ac86-11e9-96cc-2cbdc1a808fb.png">

However, I don't think it fixes the original issue, which was that when you first load the map, it doesn't let you pan without bouncing you back.  There are two things going on here: on first load (if there are no URL params), the map is given an initial display bounds of `SAN_FRANCISCO_BOUNDS`; this just tells it what bounds to display and it happens [here](https://github.com/cestastanford/imaginedsf/blob/master/static-src/scripts/maps/components/map-canvas.vue#L238).  If you follow that getter into the store, you can even see where that value comes from under different circumstances.  However, it's a separate step that restricts the map to only allow panning inside those bounds: that happens [here](https://github.com/cestastanford/imaginedsf/blob/master/static-src/scripts/maps/components/map-canvas.vue#L237) in the preceding line.  I think the best situation would be if the map's initial bounds was the updated `SAN_FRANCISCO_BOUNDS` that you have here, as it is now, but if the panning restriction used a wider bounds, perhaps double the latitude and longitude range, but still centered on San Francisco.  This would give the user the flexibility to pan around while at the top zoom level if they want while still starting the map close enough to San Francisco to highlight it and preventing them from panning way far away.

If that makes sense and you're in agreement, the way to implement that would be to add a second constant here for the bounds limit, set it to that less restrictive bounding box and use that constant in the `setMinZoom` call.